components/org.wso2.carbon.identity.api.user.authorized.apps/org.wso2.carbon.identity.rest.api.user.authorized.apps.v2/src/main/java/org/wso2/carbon/identity/rest/api/user/authorized/apps/v2/core/AuthorizedAppsService.java

```diff
@@ -222,6 +227,81 @@ public AuthorizedAppDTO listUserAuthorizedAppsByAppId(User user, String applicat
         return authorizedAppDTOS;
     }
 
+    /**
+     * Delete issued tokens for a given application id.
+     *
+     * @param applicationId Application Id.
+     */
+    public void deleteIssuedTokensByAppId(String applicationId) {
+
+        String tenantDomain = ContextLoader.getTenantDomainFromContext();
+        InboundAuthenticationRequestConfig reqConfig = getInboundAuthRequestConfig(applicationId, OAUTH2, tenantDomain);
+        String clientId = reqConfig.getInboundAuthKey();
+
+        OAuthAppRevocationRequestDTO oAuthAppRevocationRequestDTO = new OAuthAppRevocationRequestDTO();
+        oAuthAppRevocationRequestDTO.setApplicationName(applicationId);
+        oAuthAppRevocationRequestDTO.setConsumerKey(clientId);
+        oAuthAppRevocationRequestDTO.setTenantDomain(tenantDomain);
+        try {
+            OAuthRevocationResponseDTO oAuthRevocationResponseDTO =
+                    oAuthAdminService.revokeIssuedTokensByApplication(oAuthAppRevocationRequestDTO);
+            if (!oAuthRevocationResponseDTO.isError()) {
+                //TODO: Handle
```

Comment:
Is this `todo` intentional?

Reply:
Removed since that error wouldn't occur.  [892099c](https://github.com/wso2/identity-api-user/pull/129/commits/892099c6c5888c3333cacd8be25ef6a8adccda93)